packages/components/src/navigation/styles/navigation-styles.tsx

```diff
@@ -134,11 +134,11 @@ export const ItemBaseUI = styled.li`
 
 	&.is-active {
 		background-color: ${ COLORS.theme.accent };
-		color: ${ COLORS.white };
+		color: ${ COLORS.theme.accentInverted };
 
 		> button,
 		> a {
-			color: ${ COLORS.white };
+			color: ${ COLORS.theme.accentInverted };
```

Comment:
Still works well, but hover color still doesn't work well:

![Screenshot 2024-12-09 at 11 37 02](https://github.com/user-attachments/assets/90ef888b-5e41-4812-881a-8d72516855b7)

![Screenshot 2024-12-09 at 11 44 16](https://github.com/user-attachments/assets/da307533-2129-4f20-885e-61c9292507bc)


Should be fixable with something like:

```tsx
diff --git a/packages/components/src/navigation/styles/navigation-styles.tsx b/packages/components/src/navigation/styles/navigation-styles.tsx
index 0cde6314685..8894065f66f 100644
--- a/packages/components/src/navigation/styles/navigation-styles.tsx
+++ b/packages/components/src/navigation/styles/navigation-styles.tsx
@@ -137,6 +137,7 @@ export const ItemBaseUI = styled.li`
                color: ${ COLORS.theme.accentInverted };

                > button,
+               > .components-button:hover,
                > a {
                        color: ${ COLORS.theme.accentInverted };
                        opacity: 1;
```

Reply:
Seems to be working well, here's a PR: #67732